changelog.md

```diff
@@ -55,6 +55,7 @@
 * [3028](https://github.com/zeta-chain/node/pull/3028) - whitelist connection gater
 * [3019](https://github.com/zeta-chain/node/pull/3019) - add ditribute functions to staking precompile
 * [3020](https://github.com/zeta-chain/node/pull/3020) - add support for TON withdrawals
+* [3088](https://github.com/zeta-chain/node/pull/3088) - add functions to check and withdraw delegator rewards
```

Comment:
```suggestion
* [3088](https://github.com/zeta-chain/node/pull/3088) - add functions to check and withdraw delegator zrc20 rewards
```

Reply:
Added in 2b130fdc0f29a4d958d599d633c42f3987a6e7b5